src/Microsoft.IdentityModel.Tokens/EventBasedLRUCache.cs

```diff
@@ -408,24 +400,6 @@ private DateTime SetTaskEndTime()
             return DateTime.UtcNow.AddSeconds(EventQueueTaskExecutionTimeInSeconds);
         }
 
-        /// <summary>
-        /// Pause the timer.
-        /// </summary>
```

Comment:
I think I'm missing something here... why were the timer related methods removed?